### PR TITLE
Add support for Value.New().

### PR DESCRIPTION
**Pull request by rfw**

### CODE REVIEW

File: v8_c_bridge.cc [hunk 1]
```diff
@@ -421,6 +421,39 @@ ValueErrorPair v8_Value_Call(ContextPtr ctxptr,
...
+    return (ValueErrorPair){nullptr, DupString("Not a function")};
+  }
+  v8::Local<v8::Function> func = v8::Local<v8::Function>::Cast(func_val);
+
+  v8::Local<v8::Value>* argv = new v8::Local<v8::Value>[argc];
+  for (int i = 0; i < argc; i++) {
+    argv[i] = static_cast<Value*>(argvptr[i])->Get(isolate);
+  }
+
+  v8::MaybeLocal<v8::Object> result = func->NewInstance(ctx, argc, argv);
```
Comment (augustoroman) [inline v8_c_bridge.cc]: What happens if an exception is thrown?  Do you need a try/catch here?  Could you add a test along those lines?

File: v8_c_bridge.cc [hunk 2]
```diff
@@ -421,6 +421,39 @@ ValueErrorPair v8_Value_Call(ContextPtr ctxptr,
...
+
+  v8::Local<v8::Value> func_val = static_cast<Value*>(funcptr)->Get(isolate);
+  if (!func_val->IsFunction()) {
+    return (ValueErrorPair){nullptr, DupString("Not a function")};
+  }
+  v8::Local<v8::Function> func = v8::Local<v8::Function>::Cast(func_val);
+
+  v8::Local<v8::Value>* argv = new v8::Local<v8::Value>[argc];
+  for (int i = 0; i < argc; i++) {
+    argv[i] = static_cast<Value*>(argvptr[i])->Get(isolate);
```
Comment (augustoroman) [inline v8_c_bridge.cc]: I see that this is what I did in v8_Value_Call -- but out of curiosity, can you use argvptr directly?  It's a list of values, essentially, but in the wrapper classes.  My build is broken at the moment so I can't test it out myself.

Comment (rfw) [inline v8_c_bridge.cc]: I don't think so; looks like NewInstance() takes v8::Local<v8::Value> for argv only, I don't think v8::Persistent<v8::Value> can be substituted (due to GC lifetime?).

File: v8_c_bridge.cc (hunk 1 above)
Comment (rfw) [inline v8_c_bridge.cc]: Added a test -- looks like the v8::TryCatch in the scope handles it.

**Pull request merged**